app/config/services.yml

```diff
@@ -44,3 +44,9 @@ services:
 
     # needed for the localizeddate Twig filter
     Twig\Extensions\IntlExtension: ~
+
+    # the slugger service needs a public alias for getting it from
+    # the container when loading doctrine fixtures
+    slugger:
```

Comment:
Whatever solution is good to me. I only choose this as it requires less diff, and the `AppBundle\Utils\Slugger` service does not really need to be public. It's only the doctrine fixtures that use the container as a service locator, so a proper alias looks better to me.